### PR TITLE
Fix old directories used and windows/linux portability

### PR DESCRIPTION
**Pull request by noloop**
Fix of old directories that aren't in the raylib repository more ando some windows/linux portability problems.

### CODE REVIEW

File: examples/Makefile.Android [hunk 1]
```diff
@@ -159,92 +158,164 @@ all: create_temp_project_dirs \
...
-	if not exist $(PROJECT_BUILD_PATH)\bin mkdir $(PROJECT_BUILD_PATH)\bin
-	if not exist $(PROJECT_BUILD_PATH)\res mkdir $(PROJECT_BUILD_PATH)\res
-	if not exist $(PROJECT_BUILD_PATH)\res\drawable-ldpi mkdir $(PROJECT_BUILD_PATH)\res\drawable-ldpi
-	if not exist $(PROJECT_BUILD_PATH)\res\drawable-mdpi mkdir $(PROJECT_BUILD_PATH)\res\drawable-mdpi
-	if not exist $(PROJECT_BUILD_PATH)\res\drawable-hdpi mkdir $(PROJECT_BUILD_PATH)\res\drawable-hdpi
-	if not exist $(PROJECT_BUILD_PATH)\res\values mkdir $(PROJECT_BUILD_PATH)\res\values
-	if not exist $(PROJECT_BUILD_PATH)\assets mkdir $(PROJECT_BUILD_PATH)\assets
-	if not exist $(PROJECT_BUILD_PATH)\assets\$(PROJECT_RESOURCES_PATH) mkdir $(PROJECT_BUILD_PATH)\assets\$(PROJECT_RESOURCES_PATH)
-	if not exist $(PROJECT_BUILD_PATH)\obj\screens mkdir $(PROJECT_BUILD_PATH)\obj\screens
+	mkdir -p $(PROJECT_BUILD_PATH)
```
Comment (raysan5) [inline examples/Makefile.Android:161]: Did you test it on Windows CMD? It seems `-p` does not work.

File: examples/Makefile.Android [hunk 2]
```diff
@@ -159,92 +158,164 @@ all: create_temp_project_dirs \
...
 
 define create_dir
-    if not exist $(PROJECT_BUILD_PATH)\obj\$(1) mkdir $(PROJECT_BUILD_PATH)\obj\$(1)
+    mkdir -p $(PROJECT_BUILD_PATH)/obj/$(1)
 endef
     
+ifeq ($(OS),Windows_NT)
+    COPY_COMMAND ?= copy /Y
+    COPY_DIRECTORY_COMMAND = C:\Windows\System32\xcopy /Y /E /F
+    ANDROID_STRING_XML_LINE1 = ^<?xml version="1.0" encoding="utf-8"^?^>
```
Comment (raysan5) [inline examples/Makefile.Android:188]: I would prefer to avoid that many variables, the `ifeq` could just be considered inside the target.

**Comment by raysan5** [comment]
@noloop Thank you very much for the improvement! I added a couple of comments. Did you try compiling some of the examples for Android? Is the generated .apk working on real device?

File: examples/Makefile.Android (hunk 1 above)
Comment (noloop) [reply examples/Makefile.Android:161]: I forgot, mkdir -p doesn't work in windows, I'll fix this.

**Comment by noloop** [comment]
Yes, I tested in the one of examples of the core and the apk is working in my cell phone.

File: examples/Makefile.Android (hunk 2 above)
Comment (noloop) [reply examples/Makefile.Android:188]: I'm fixing this now.

File: examples/Makefile.Android (hunk 1 above)
Comment (noloop) [reply examples/Makefile.Android:181]: Already fixed.

File: examples/Makefile.Android (hunk 2 above)
Comment (noloop) [reply examples/Makefile.Android]: Already fixed.

**Comment by raysan5** [comment]
@noloop Thanks for the update! :)

**Pull request merged**